lib/matplotlib/tests/test_axes.py

```diff
@@ -6273,7 +6272,8 @@ def test_datetime_masked():
     fig, ax = plt.subplots()
     ax.plot(x, m)
     # these are the default viewlim
```

Comment:
This comment should maybe be changed a bit.

Reply:
Just removed it